tmt/utils.py

```diff
@@ -926,7 +926,9 @@ def last_run(self, workdir: Path) -> None:
             self._last_run_symlink.symlink_to(workdir)
         except FileExistsError:
             # Race when tmt runs in parallel
-            log.warning(f"Race condition, unable to save last run '{workdir}'.")
+            log.warning(
+                f"Race condition, unable to save last run '{workdir}'. "
```

Comment:
How about?

```suggestion
                f"Race condition, unable to save last run symlink to '{workdir}'. "
```

Reply:
I'd maybe drop the "Race condition" bit if it's aiming at the user, how about "Unable to mark '{workdir}' as the last run, `tmt run --last` might not pick the right run directory."? There is a race condition, but should the user care about such a fact? tmt failed to update last run symlink, that's the gist.

Reply:
Yeah, this is better, updated in ada3726d.